docs/docs-content/clusters/data-center/vmware.md

```diff
@@ -1072,6 +1049,18 @@ In addition to the default cloud account already associated with the private clo
 | **Username** | vCenter username|
 | **Password** | vCenter password|
 
+:::caution
+
+If you change the password for a user account in Vcenter, you must also change it in Palette for the same VMware cloud account. We suggest updating the passwords sequentially to avoid potentially locking Palette out of Vcenter.
```

Comment:
```suggestion
If you change the password for a user account in Vcenter, you must also change it in Palette for the same VMware cloud account. We recommend updating the passwords immediately to avoid potentially locking Palette out of Vcenter.
```
using the terms "we suggests" feels like we are trying to put some distance between us and the reader. We recommend is more accurate reflection of the message we want to convey.

Reply:
Changed.